library/ssl_tls.c

```diff
@@ -5956,6 +5970,17 @@ static unsigned char ssl_serialized_session_header[] = {
  * The order is the same as in the definition of the structure, except
  * verify_result is put before peer_cert so that all mandatory fields come
  * together in one block.
+ *
+ * In TLS/DTLS 1.3 the ticket contains the following content:
+ *
+ *  - ticket creation time ( start )
```

Comment:
We should use the RFC-like field notation from above here.

Reply:
I didn't use RFC style notation here to avoid confusing the reading with the any of the ticket descriptions in RFCs. This text is really about our internal ticket structure. I put clarifying words there

Reply:
I would prefer sticking to the RFC notation, as that is what we're using in upstream TLS - I expect pushback at the very latest once we upstream.

Reply:
I changed my mind. I updated the description to use RFC-style here as well.